clash-cores/src/Clash/Cores/Xilinx/Floating/Explicit.hs

```diff
@@ -311,6 +316,45 @@ fromS32 = withFrozenCallStack fromS32With
 -- | The default delay for conversion of @Signed 32@ to @Float@
 type FromS32DefDelay = 6
 
+-- | Compare two floating point numbers. Produces 'NaN' if any of the inputs
+-- is NaN.
```

Comment:
```suggestion
-- | Customizable floating point comparison
--
-- Produces 'NaN' if any of the inputs is NaN. Otherwise, it behaves like
-- Haskell's 'P.compare'.
--
-- Only the delay is configurable, so this function does not take a @Config@
-- argument.
```